artview/scripts/edit_radar.py

```diff
@@ -8,8 +8,7 @@
 import sys
 
 from ..core import Variable, QtWidgets
-from ..components import RadarDisplay, Menu, SelectRegion
-from ..plugins import GateFilter
+from ..components import RadarDisplay, Menu, SelectRegion, GateFilter
```

Comment:
I was mistaken, I though this was a pep8 thing. Can't find it so yo must be right!